js/load.js

```diff
@@ -0,0 +1,246 @@
+
+  infoItem = document.createElement('li');
+  infoItemLink = document.createElement('time');
+  infoItemLink.classList.add('category', 'flex', 'cross-center', 'ellipse', 'line-height', 'highlight', 'cap');
+  infoItemLink.innerHTML = work.info.year;
+  infoItem.appendChild(infoItemLink);
+  infoList.appendChild(infoItem);
+
+  const modalImage = document.createElement('img');
+  modalImage.setAttribute('src', work.image.src);
+  modalImage.setAttribute('alt', work.image.alt);
+  modalWrapper.appendChild(modalImage);
+
+  const modalBottom = document.createElement('div');
+  modalBottom.classList.add('modal-bottom', 'flex', 'flex-column');
+  modalWrapper.appendChild(modalBottom);
+
+  const modalContent = document.createElement('div');
+  modalContent.classList.add('modal-content');
+  modalBottom.appendChild(modalContent);
+
+  const modalHero = document.createElement('div');
+  modalHero.classList.add('content', 'hero');
+  modalContent.appendChild(modalHero);
+
+  const modalDescription = document.createElement('p');
+  modalDescription.innerHTML = work.description;
+  modalHero.appendChild(modalDescription);
+
+  const modalSide = document.createElement('div');
+  modalSide.classList.add('modal-side');
+  modalBottom.appendChild(modalSide);
+
+  const tagsList = document.createElement('ul');
+  tagsList.classList.add('tags', 'flex');
+  modalSide.appendChild(tagsList);
+
+  for (let i = 0; i < work.technologies.length; i += 1) {
+    const tagsListItem = document.createElement('li');
+    tagsList.appendChild(tagsListItem);
+
+    const tagsListItemLink = document.createElement('a');
+    tagsListItemLink.classList.add('line-height', 'round-borders');
+    tagsListItemLink.innerHTML = work.technologies[i];
+    tagsListItemLink.href = '#';
+    tagsListItem.appendChild(tagsListItemLink);
+  }
+
+  const modalFooter = document.createElement('div');
+  modalFooter.classList.add('modal-footer', 'flex', 'space-between');
+  modalSide.appendChild(modalFooter);
+
+  for (let i = 0; i < 2; i += 1) {
+    const modalFooterBtn = document.createElement('button');
+    modalFooterBtn.classList.add('bg', 'round-borders', 'cap');
+    modalFooterBtn.type = 'button';
+    const textSpan = document.createElement('span');
+    textSpan.innerHTML = work.buttons[Object.keys(work.buttons)[i]].text;
+    modalFooterBtn.appendChild(textSpan);
+    const imgSpan = document.createElement('img');
+    imgSpan.setAttribute('src', work.buttons[Object.keys(work.buttons)[i]].imgSrc);
+    imgSpan.setAttribute('alt', work.buttons[Object.keys(work.buttons)[i]].text);
+    modalFooterBtn.appendChild(imgSpan);
+    modalFooter.appendChild(modalFooterBtn);
+  }
+
+  document.body.appendChild(modal);
+}
+
+const seeButtons = document.querySelectorAll('.see');
+
+for (let i = 0; i < seeButtons.length; i += 1) {
+  seeButtons[i].addEventListener('click', () => {
+    displayPopup(works[i]);
+  });
+}
+
+// eslint-disable-next-line no-plusplus
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them. you can fix this by simply use `i += 1` instead of `i++`